src/docs/cookbook/networking/delete-data.md

```diff
@@ -9,18 +9,20 @@ next:
   path: /docs/cookbook/networking/authenticated-requests
 ---
 
-You'll learn how to delete data on internet using the `http` package.
+In this recipe, you'll learn how to delete data over
+the internet using the `http` package.
 
 This recipe uses the following steps:
 
-  1. Add the `http` package.
-  2. Delete data on the server.
-  3. Update the screen.
+  1. Add the `http` package
+  2. Delete data on the server
+  3. Update the screen
```

Comment:
See above re: punctuation.

Reply:
These aren't all full sentences, so you don't use punctuation in that case. Mostly, though, these are also the names of headers, so I've left off the punctuation.